readme.md

```diff
@@ -72,11 +72,11 @@ running `dependency-check package.json --no-default-entries --entry tests.js` wo
 
 ### --extensions, -e
 
-running `dependency-check ./package.json -e js,jsx` will resolve extensionless require paths to both `.js` and `.jsx` paths. Defaults to just `.js`
+running `dependency-check ./package.json -e js,jsx:detective-es6` will resolve extensionless require paths to both `.js` and `.jsx` paths and parse using [`precinct`](https://www.npmjs.com/package/precinct). Defaults to `.js` using [`detective`](https://www.npmjs.com/package/detective).
```

Comment:
Same here – I would like to keep the simple  `-e` example and make the advanced `-e` example so advanced that one see how one goes about adding multiple detective parsers at once.